Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/05-PersistingSyncUps/PersistingSyncUps.tutorial

```diff
@@ -16,6 +16,8 @@
         Go back to the SyncUpsList.swift file, and start by applying the [`@Shared`](<doc:Shared>)
         property wrapper to the `syncUps` field.
         
+        > Note: Our changes will not compile until step 3.
```

Comment:
Just a suggestion to throw this in, because often when I go through enough steps in a tutorial without code compiling, I end up wondering if I missed something in a step, which distracts from the tutorial itself.